assets/js/templates/snippet/image.html

```diff
@@ -0,0 +1,6 @@
+<!-- File Button -->
+<div class="control-group">
+  <label class="control-label"
+         for="<%= field_guid %>"><%= label %></label>
+  <button id="<%= field_guid %>" name="<%= field_guid %>">Upload</button>
```

Comment:
Do we want the 'btn' class (or any others) like here: https://github.com/iFixit/Bootstrap-Form-Builder/blob/master/assets/js/templates/snippet/button.html#L5?